src/index.js

```diff
@@ -0,0 +1,158 @@
+/*
+ * decaffeinate suggestions:
+ * DS001: Remove Babel/TypeScript constructor workaround
+ * DS101: Remove unnecessary use of Array.from
+ * DS102: Remove unnecessary code created because of implicit returns
+ * DS206: Consider reworking classes to avoid initClass
+ * DS207: Consider shorter variations of null checks
+ * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
+ */
+const Fs    = require('fs');
+const Path  = require('path');
+const Hubot = require('hubot/es2015');
+
+process.setMaxListeners(0);
+
+class MockResponse extends Hubot.Response {
+  sendPrivate(...strings) {
```

Comment:
do we still need this method? Hubot.Response doesn't have it

Reply:
Wow, good catch.

I had thought this method implemented in `Hubot.Response` https://github.com/mtsmfm/hubot-test-helper/pull/6

But actually it seems that it comes from https://github.com/nandub/hubot-irc/blob/7c35ef19b2eb7ea5f3284b3d5af897a747d56309/src/irc.coffee#L6

But I don't want to introduce breaking change so let's leave it for now.